lib/config_read.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require 'fluent/config/v1_parser'
+require 'optparse'
+
+# Accepts a file path and prints out parsed version
+class ConfigFile
```

Comment:
```suggestion
class ConfigParser
```

Reply:
Recommend making the class name as what this tool does, also rename the file name (`config_read.rb`) to the corresponding class name.

An alternative I can think about is `ConfigReader`.

Reply:
I like ConfigParser so I'll make that change, thanks!